microsoft-edge/extensions-chromium/publish/verify-microsoft-edge-program.md

```diff
@@ -0,0 +1,90 @@
+
+
+<!-- ====================================================================== -->
+## Checking your verification status
+
+You can check verification status at Partner Center in **[Account Settings | Legal Info](https://partner.microsoft.com/dashboard/account/v3/organization/legalinfo#developer)**, where you can see:
+
+1. One or more tabs such as **Partner**, **Developer**, or **Reseller**, depending on the programs in which your organization is enrolled.
+1. Your verification status: **Pending**, **Accepted**, or **Rejected**, with a status icon.
+1. The **Legal business profile** progress bar with an information icon you can select to get more information.
+
+
+<!-- ====================================================================== -->
+## Verification status
+
+There are three possible results when you check your verification status:
+
+*   **Authorized/Accepted**: The information you submitted was verified, and you're notified of your acceptance into the program. No further action is required.
+
+*   **Pending**: The verification process has started but isn't complete. No action is required. You can monitor verification status at Account Settings | Legal Info. Verification usually takes 3-5 business days.
+
+*   **Action Required/Rejected**: The information you submitted couldn't be verified. The reason and instructions for [how to appeal](#appealing-a-rejected-application) appear in the Account Verification pane.
+
+
+<!-- ====================================================================== -->
+## Appealing a rejected application
+
+To appeal a rejected application:
+
+1. At **[Account Settings | Legal Info](https://partner.microsoft.com/dashboard/account/v3/organization/legalinfo#developer)**, select **Fix now**.
+1. In **Account Verification**, from **Select document type to upload**, select the type of document that you want to upload for verification.
+1. In **Enter your comment**, you can enter more information about your appeal.
+1. Select **Upload**.
+
+The amount of time required to review an appeal varies. You can return to **[Account Settings | Legal Info](https://partner.microsoft.com/dashboard/account/v3/organization/legalinfo#developer)** to check verification status at any time. Verification status is **Pending** during the review.
+
+![Account Settings | Legal Info](media/account-settings-legal-info-microsoft-edge-partner-center.png)
```

Comment:
@ShashankMG123 commented: "Account settings image is not very clear can we get another image?"

Confirmed, right-click > "Open image in new tab" doesn't show more detail; the png lacks detail.

Rendered: https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/verify-microsoft-edge-program?branch=pr-en-us-1963#appealing-a-rejected-application